chapters/packages.tex

```diff
@@ -321,3 +321,135 @@ \section{External Resources}\label{external-resources}
 \filename{modelica://Modelica/Mechanics/C.jpg} is illegal.
 The references \filename{modelica://Modelica.Mechanics/C.jpg} and \filename{modelica://Modelica/C.jpg} must also refer to two distinct resources.
 \end{example}
+
+\section{Multilingual Descriptions}\label{multilingual-descriptions}
+
+\begin{nonnormative}
+Descriptive texts in a model or library are usually formulated in English.
+To support users who are not so familiar with this, a Modelica tool can support the translation of these texts into any language.
```

Comment:
This sounds too colloquial to me.

Additionally, a Modelica tool should not only support the translation (in the sense of translation process), but a Modelica IDE should display the localized/translated Modelica package.

Reply:
I hope the new formulation is good enough.